src/fermilib/utils/_sparse_tools.py

```diff
@@ -300,7 +301,7 @@ def expectation(sparse_operator, state):
     """Compute expectation value of operator with a state.
 
     Args:
-        state_vector: scipy.sparse.csc vector representing a pure state,
+        state: scipy.sparse.csc vector representing a pure state,
```

Comment:
The whole point of this function is to take these expectation values in polynomial time with respect to N. Why would you give a scipy.sparse.csc vector as input instead of a bit string?